File: 프로그래머스/0/181902. 문자 개수 세기/문자 개수 세기.js
function solution(my_string) {
    var answer = new Array(52).fill(0);
    for(let i = 0; i<my_string.length; i++){
        if(my_string.charCodeAt(i) >= 97 && my_string.charCodeAt(i) <= 122) answer[my_string.charCodeAt(i) - 71]++;
        else answer[my_string.charCodeAt(i) - 65]++;
    }
    return answer;
}

File: 프로그래머스/0/181902. 문자 개수 세기/문자 개수 세기.py
def solution(my_string):
    strAlp = "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz"
    answer = [0]*52
    for i in my_string:
        answer[strAlp.index(i)] += 1
    return answer